kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/BatchRestApi.java

```diff
@@ -47,9 +47,12 @@ public Batch getBatchById(String batchId) {
     return this.getClient().get(path, null, Batch.class, client.getAuthHeader());
   }
 
-  public GetBatchesResponse listBatches(String batchType, int from, int size) {
+  public GetBatchesResponse listBatches(
+      String batchType, String batchUser, String batchState, int from, int size) {
     Map<String, Object> params = new HashMap<>();
     params.put("batchType", batchType);
+    params.put("batchUser", batchUser);
+    params.put("batchState", batchState);
```

Comment:
how about supporting filter by createTime and endTime ?

Reply:
seems not needed, now the createTime and endTime are all LongType and it is difficult to filter now.

Reply:
let me have a try